feature/isis/otg_tests/weighted_ecmp_test/weighted_ecmp_test.go

```diff
@@ -200,7 +214,30 @@ func TestWeightedECMPForISIS(t *testing.T) {
 	}
 	p3 := dut.Port(t, "port3")
 	gnmi.Await(t, dut, gnmi.OC().Interface(p3.Name()).OperStatus().State(), time.Minute*2, oc.Interface_OperStatus_DOWN)
+
+	if deviations.WecmpAutoUnsupported(dut) {
+		var weight string
+		switch dut.Vendor() {
+		case ondatra.CISCO:
+			weight = " router isis DEFAULT \n interface Bundle-Ether 2 \n address-family ipv4 unicast \n weight 200 \n address-family ipv6 unicast \n weight 200 \n ! \n interface Bundle-Ether 3 \n address-family ipv4 unicast \n weight 400 \n address-family ipv6 unicast \n weight 400 \n ! \n interface Bundle-Ether 4 \n address-family ipv4 unicast \n weight 400 \n address-family ipv6 unicast \n weight 400 \n"
+		default:
+			t.Fatalf("Unsupported vendor %s for deviation 'WecmpAutoUnsupported'", dut.Vendor())
+		}
+		helpers.GnmiCLIConfig(t, dut, weight)
+	}
+
 	top.Flows().Clear()
+	flows = configureFlows(t, top, ate1AdvV4, ate1AdvV6, ate2AdvV4, ate2AdvV6)
+	ate.OTG().PushConfig(t, top)
+	ate.OTG().StartProtocols(t)
+	VerifyISISTelemetry(t, dut, []string{"Bundle-Ether1", "Bundle-Ether2", "Bundle-Ether3", "Bundle-Ether4"})
+	cfgplugins.VerifyDUTBGPEstablished(t, dut)
+	// wait for BGP to converge
+	for _, agg := range []*aggPortData{agg1, agg2} {
+		bgpPath := gnmi.OC().NetworkInstance(deviations.DefaultNetworkInstance(dut)).Protocol(oc.PolicyTypes_INSTALL_PROTOCOL_TYPE_BGP, "BGP").Bgp()
+		gnmi.Await(t, dut, bgpPath.Neighbor(agg.ateLoopbackV4).SessionState().State(), 2*time.Minute, oc.Bgp_Neighbor_SessionState_ESTABLISHED)
+		gnmi.Await(t, dut, bgpPath.Neighbor(agg.ateLoopbackV6).SessionState().State(), 2*time.Minute, oc.Bgp_Neighbor_SessionState_ESTABLISHED)
+	}
```

Comment:
Its taking time for the above change in config to take effect and the below traffic loss verification at line 246 fails , please add a delay after starttraffic at line 242